### ISSUE TITLE
CORE-AAM should suggest when to propagate readonly/editable state to gridcells

### ISSUE THREAD

**Issue by aleventhal** [w3c/aria#572]
For both treegrid and grid, if @aria-readonly is specified as true or false (not left out), then appropriate readonly/editable states for the given API should be propagated to each gridcell descendant that doesn't override aria-readonly with its own state.

**Comment by joanmarie** [comment]
@aleventhal: There is already a statement under [`aria-readonly="true"`](https://rawgit.com/w3c/aria/master/core-aam/core-aam.html#ariaReadonlyTrue). So:

* Is that statement acceptable to you as-is?
* I assume you would like a parallel statement under `aria-readonly="false"`. Is that correct?
* Do you think it needs to be copied under the `grid` and `treegrid` role mappings in addition to the above, or is the above sufficient?

**Comment by joanmarie** [comment]
@aleventhal: I gave it some more thought. Will this new table entry address your concerns (both the approach and the actual statement): https://rawgit.com/w3c/aria/master/core-aam/core-aam.html#ariaReadonlyUnspecifiedOnGridcell

**Comment by aleventhal** [comment]
@joanmarie that's great, thanks!

**Comment by joanmarie** [comment]
Thanks! Then closing this issue has having been addressed.
